packages/react-native-client/src/hooks/useScreenShare.ts

```diff
@@ -108,6 +91,29 @@ export function useScreenShare() {
     return 'denied';
   }, []);
 
+  const toggleScreenShare = useCallback(
+    async (screenShareOptions: Partial<ScreenShareOptions> = {}) => {
+      if (Platform.OS === 'android' && !isScreenShareOn) {
+        (await handleScreenSharePermission()) == 'granted' &&
```

Comment:
what if user doesn't grant permission?

Reply:
A warning will be emitted to the developer: "No permission to start screen share, call handleScreenSharePermission first."

Reply:
Shouldn't developer be able to handle this gracefully, like displaying a modal to the user "hey, you didn't grant the permission to screen share, go to settings if you change your mind" or something like that?

Reply:
`toggleScreenShare` is a promise that the developer can catch. However, I can see now this this could be handled better with an error code rather then a message.

Reply:
Ok, I checked that this is a class, so the exception code is `ERR_MISSING_SCREEN_SHARE_PERMISSION` so everything should be fine.